algorithms/scaling/scaling_library.py

```diff
@@ -381,7 +381,11 @@ def determine_best_unit_cell(experiments):
     experiments."""
     uc_params = [flex.double() for i in range(6)]
     for exp in experiments:
-        for i, p in enumerate(exp.crystal.get_unit_cell().parameters()):
+        if exp.crystal.get_recalculated_unit_cell() is not None:
+            unit_cell = exp.crystal.get_recalculated_unit_cell()
+        else:
+            unit_cell = exp.crystal.get_unit_cell()
```

Comment:
maybe
```suggestion
        unit_cell = exp.crystal.get_recalculated_unit_cell() or exp.crystal.get_unit_cell()
```
but you iirc you don't like that syntax.

Reply:
I think in this case it is probably justified as it is a simple enough assignment and makes the code much shorter.